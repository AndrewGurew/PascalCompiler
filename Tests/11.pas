a = array[1001..12123] of integer;