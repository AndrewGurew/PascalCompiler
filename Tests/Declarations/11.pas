type
a = integer;
b = double;
var 
c: integer;