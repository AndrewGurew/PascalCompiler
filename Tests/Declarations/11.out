Declaration
 ⎬b - Double(Type)
 ⎬a - Integer(Type)
 ⎬c - Integer(Var)